showcase/src/app/app.component.html

```diff
@@ -55,6 +55,9 @@
 			<a mat-list-item uiSref="demo-slider" uiSrefActive="active">
 				<span matLine>Slider</span>
 			</a>
+			<a mat-list-item uiSref="stark-footer" uiSrefActive="active">
```

Comment:
The state name is now `demo-stark-footer`...  cause the app fails now when clicking on this menu item